src/Cli/dotnet/NugetPackageInstaller/NuGetPackageDownloader.cs

```diff
@@ -72,7 +72,7 @@ public NuGetPackageDownloader(DirectoryPath packageInstallDir, IFilePermissionSe
 
             if (resource == null)
```

Comment:
Do the calls to `GetLatestVersionInternalAsync` and `GetPackageMetadataAsync` succeed if there is no package found?  I would have expected to need to check for an error there.